docs/standard-library/multimap-class.md

```diff
@@ -30,7 +30,7 @@ The element data type to be stored in the multimap.
 *`Traits`*\
 The type that provides a function object that can compare two element values as sort keys to determine their relative order in the multimap. The binary predicate `less<Key>` is the default value.
 
-In C++14 you can enable heterogeneous lookup by specifying the `std::less<>` or `std::greater<>` predicate that has no type parameters. For more information, see [Heterogeneous Lookup in Associative Containers](../standard-library/stl-containers.md#heterogeneous-lookup-in-associative-containers-c14)
```

Comment:
Are you sure about removing the -c14? 
When I look at the link in the browser, its #heterogeneous-lookup-in-associative-containers-c14

Reply:
Since I added a custom anchor `heterogeneous-lookup-in-associative-containers`, that will be used instead (i.e. without the "-c14"). The "-c14" is present in the browser as we don't have a custom anchor, and the default one is being used. I have cross checked with other instances and verified that doing `#custom-anchor` works. Hope that clears it up.